client/query/query.go

```diff
@@ -74,19 +75,37 @@ func getLedgerBalanceFromState(latest state.State) LedgerChannelBalance {
 	}
 }
 
-// GetPaymentChannelInfo returns the PaymentChannelInfo for the given channel
-// It does this by querying the provided store and voucher manager
-func GetPaymentChannelInfo(id types.Destination, store store.Store, vm *payments.VoucherManager) (PaymentChannelInfo, error) {
+func isVirtualFundObjective(id types.Destination, store store.Store) (*virtualfund.Objective, bool) {
+	// This is slightly awkward but if the virtual defunding objective is complete it won't come back if we query by channel id
+	// We manually construct the objective id and query by that
+	virtualFundId := protocols.ObjectiveId(virtualfund.ObjectivePrefix + id.String())
+	o, err := store.GetObjectiveById(virtualFundId)
+	if err != nil {
+		return nil, false
+	}
+	return o.(*virtualfund.Objective), true
+}
+
+func isVirtualDefundObjective(id types.Destination, store store.Store) (*virtualdefund.Objective, bool) {
```

Comment:
```suggestion
func isVirtualDefundObjective(id types.Destination, store store.Store) *virtualdefund.Objective {
```
Do we need the `bool` return value? It seems like the two return values are coupled. If the result is nil, the second return value is set to false. If the result is not nil, the second return value is set to true.

Reply:
This is mostly syntactical sugar that mimicks how casting from an interface works in go: `v, ok = i.(T)`. I personally prefer having the extra bool value, even if it's not strictly necessary ,as I think it makes things slightly neater. However I don't feel super strongly about it!

Reply:
Gotcha. Maybe the method name is throwing me off. This method seems different than a cast. This method is more of of `getVirtualDefundObjectiveIfExists`. I don't feel strongly about the return type either.